pavelib/utils/test/suites/nose_suite.py

```diff
@@ -157,6 +157,7 @@ def _default_test_id(self):
 
         if self.root == 'lms':
             default_test_id += " {system}/tests.py"
+            default_test_id += " openedx/core/lib/*"
```

Comment:
@jzoldak I think you mentioned adding these to system tests (vs lms tests).  Can you confirm?